src/Uno.UI/UI/Xaml/Media/VisualTreeHelper.cs

```diff
@@ -91,7 +86,7 @@ public static IEnumerable<UIElement> FindElementsInHostCoordinates(Point interse
 		private static bool IsElementIntersecting(Point intersectingPoint, UIElement uiElement)
 		{
 			GeneralTransform transformToRoot = uiElement.TransformToVisual(null);
-			var target = transformToRoot.TransformBounds(uiElement.LayoutSlot);
+			var target = transformToRoot.TransformBounds(new Rect(0, 0, uiElement.ActualSize.X, uiElement.ActualSize.Y));
```

Comment:
Looking at the code, your fix is right .... but the whole `FindElementsInHostCoordinates` method needs to be refactor : we are navigating the whole tree top->down, but for EACH element we are using the `TransformToVisual` which will navigate down->top 😨 https://github.com/unoplatform/uno/issues/13281

Reply:
I revert this fix because this is not unnecessary for this PR. I reviewed the code and realized that I had used the `FindElementsInHostCoordinates` in my previous wrong fix.